frontend/src/features/entity_selector/EntitySelector.tsx

```diff
@@ -267,12 +330,15 @@ const EntitySelectorInnerAuth = ({
             </Grid>
             <Grid item>
               <AutoEntityButton
+                htmlId={`auto-suggestion-${alignment}`}
                 disabled={loading}
                 currentUid={uid}
                 otherUid={otherUid}
-                onClick={() => {
+                onClick={async () => {
                   setLoading(true);
                   setInputValue('');
+                  setSlideIn(false);
+                  await wait(ENTITY_CARD_SWIPE_TIMEOUT + 8);
```

Comment:
The `wait` function ensures the swipe up animation is displayed before displaying a new video, but using `setTimeout` might not be very elegant. What do you think?

An alternative could be to move the logic of the `askNewVideo` callback from the component `AutoEntityButton` to `EntitySelector`. This way `setTimeout` won't be required anymore as `EntitySelector` will be able to easily trigger the video refresh after the swipe up animation. Therefore, the component `AutoEntityButton` will be a simple button that requires its parent to implement the refresh logic.